domains/certificates/Query.API/API/Configurations/DatasyncOptions.cs

```diff
@@ -9,6 +9,9 @@ public class DatasyncOptions
 
     [Required]
     public string Url { get; set; } = string.Empty;
+
+    [Required]
+    public bool Disabled { get; set; } = false;
```

Comment:
What is the scenario where we want to disable the datasyncsyncer?

Reply:
We need to disable it when switching to the new 'flow' to allow old messages to finish before switching. It's only ment to be disabled for a short amount of time, this time.